packages/jest-cli/src/cli/args.ts

```diff
@@ -503,6 +510,13 @@ export const options = {
       'rare.',
     type: 'boolean',
   },
+  runProjects: {
+    description:
+      'Run only the tests of the specified projects.' +
+      'Jest uses the attribute `name` in the configuration to identify each project.',
```

Comment:
Shouldn't it be:
```suggestion
      'Jest uses the attribute `displayName` in the configuration to identify each project.',
```
?

Reply:
Nice catch. Thank you.